src/components/event/CreateEvent.js

```diff
@@ -177,8 +177,8 @@ class CreateEvent extends Component {
 			if (this.props.title === "Create Event") { createEvent(type, committee, name, description, date, startTime, endTime, location, points) }
 			else {
 				editEvent(type, committee, name, description, date, `${startTimeComp[0]}:${startTimeComp[1]}:${startTimeComp[2]}`, `${endTimeComp[0]}:${endTimeComp[1]}:${endTimeComp[2]}`, location, points, eventID);
-				this.props.startTimeChanged(this.convertHour(`${startTimeComp[0]}:${startTimeComp[1]}:${startTimeComp[2]}`));
-				this.props.endTimeChanged(this.convertHour(`${endTimeComp[0]}:${endTimeComp[1]}:${endTimeComp[2]}`));
+				// this.props.startTimeChanged(this.convertHour(`${startTimeComp[0]}:${startTimeComp[1]}:${startTimeComp[2]}`));
+				// this.props.endTimeChanged(this.convertHour(`${endTimeComp[0]}:${endTimeComp[1]}:${endTimeComp[2]}`));
```

Comment:
These lines are so long I died, but since they weren't changed in this PR, I guess they don't need to be fixed now

Reply:
wait i'm sorry this file needs to be removed